packages/types/src/codec/createType.ts

```diff
@@ -315,11 +320,13 @@ function initType<T extends Codec = Codec> (Type: Constructor<T>, value?: any, i
   }
 }
 
-export default function createType<T extends Codec = Codec> (type: Text | string, value?: any, isPedantic?: boolean): T {
-  // l.debug(() => ['createType', { type, value }]);
-
+export default function createType<T extends Codec = Codec, K extends Text | string = Text | string> (
```

Comment:
I didn't remove Text yet, could do it now, could do it later when we cater for https://github.com/polkadot-js/api/issues/1177#issuecomment-514588305

Reply:
I'll do this in a subsequent PR, this one is big enough